examples/rest_ex.py

```diff
@@ -51,21 +51,25 @@ def t(self):
         with self.rest("GET", "http://example.com:42/", json={"foo": 1}) as resp:
             pass
 
+
 class RestUserThatLooksAtErrors(RestUser):
     abstract = True
+
     @contextmanager
-    def rest(self, method, url, headers={}, **kwargs) -> ResponseContextManager:
+    def rest(self, method, url, **kwargs) -> ResponseContextManager:
         extra_headers = {"my_header": "my_value"}
-        with super().rest(method, url, headers={**headers, **extra_headers}, **kwargs) as resp:
-            # 
+        with super().rest(method, url, headers=extra_headers, **kwargs) as resp:
+            #
             if "error" in resp.js and resp.js["error"] is not None:
-                resp.failure(resp.js["error"])
+                # pylint thinks this is a FASTUser, and not a ResponseContextManager
+                resp.failure(resp.js["error"])  # pylint: disable=E1101
             yield resp
 
+
 class MyOtherRestUser(RestUserThatLooksAtErrors):
     @task
     def t(self):
-        with self.rest("GET", "/") as resp:
+        with self.rest("GET", "/") as resp:  # pylint: disable=W0612
```

Comment:
This is weird and should not be required.. can you investigate?

Reply:
Looks like it just needed the hint on the object type like rest.py did.

Reply:
never mind the above comment - that was with respect to the other pylint override - which i have fixed now.

this one is needed because nothing uses resp - so its just tagged as an unused variable. The same error shows up if you remove lines 28-40. 

I can remove this if I add some check/test/use of resp - but the pylint disable is pretty minor.

Reply:
Either name it _resp or set dummy-variables-rgx in .pylintrc to match resp (latter preferred). This is an example so it should have as little noice as possible.